### PR TITLE
Jb t2d slim solo hcs

### PR DESCRIPTION
**Pull request by jbloom22**

### CODE REVIEW

File: input.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (cseed) [inline input.json]: Can't leave this in the toplevel.

File: input2.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (cseed) [inline input2.json]: Same.

File: src/main/scala/org/broadinstitute/hail/driver/LinearRegressionOnHcsCommand.scala [hunk 3]
```diff
@@ -0,0 +1,118 @@
+package org.broadinstitute.hail.driver
+
+/*
```
Comment (cseed) [inline src/main/scala/org/broadinstitute/hail/driver/LinearRegressionOnHcsCommand.scala]: This is entirely commented out.  Remove?

File: src/test/scala/org/broadinstitute/hail/methods/LinearRegressionSuite.scala [hunk 4]
```diff
@@ -392,4 +392,103 @@ class LinearRegressionSuite extends SparkSuite {
         "-c", "sa.cov.Cov1,sa.cov.Cov2"))
     }
   }
+
+  /*
```
Comment (cseed) [inline src/test/scala/org/broadinstitute/hail/methods/LinearRegressionSuite.scala]: Same.

File: src/test/scala/org/broadinstitute/hail/variant/HardCallSetSuite.scala [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    assert(hcs.nDenseVariants == 5)
+    assert(hcs.nSamples == 6)
+    assert(hcs.sampleIds == filtVds.sampleIds)
+    assert(hcs.sampleIds == filtVds.sampleIds)
+
+    assert(hcs.df.groupBy("contig", "block").count().first().getLong(2) == 7)
+
+    assert(hcs.capNVariantsPerBlock(3).df.groupBy("contig", "block").count().first().getLong(2) == 3)
+
+    // hcs.writeNVariantsPerBlock("nVariantsPerBlock.tsv")
```
Comment (cseed) [inline src/test/scala/org/broadinstitute/hail/variant/HardCallSetSuite.scala]: Same.

**Comment by cseed** [comment]
Couple of things:

You left in a bunch of commented out blocks that need to get cleaned up.

You need to squash the history.  That might be hard.  The solution is to merge the current master, create a diff between the resulting version and master, and apply that to a fresh copy of master and then PR against that (or force this branch to the resulting commit).

Finally, I'd like you to break out the gradle dependencies as a separate PR first.  I need that for the seqr stuff ASAP.

Thanks!

**Comment by cseed** [comment]
I can look at this so we can start to fold it in.  I made some cleanup comments (checked in json files, blocks of commented out code).  Also, it duplicates the gradle changes in the master so you'll want to rebase and, finally, you need to clean up the commit history.  No hurry.

**Comment by jbloom22** [comment]
continuing at PR #517

**Pull request closed without merging**